kontron-imx8mm.coffee

```diff
@@ -19,20 +19,20 @@ module.exports =
 
 	instructions: commonImg.instructions
 	# gettingStartedLink:
-	# 	windows: 'https://www.balena.io/docs/learn/getting-started/kontron-mx8mm/nodejs/'
-	# 	osx: 'https://www.balena.io/docs/learn/getting-started/kontron-mx8mm/nodejs/'
-	# 	linux: 'https://www.balena.io/docs/learn/getting-started/kontron-mx8mm/nodejs/'
+	# 	windows: 'https://www.balena.io/docs/learn/getting-started/kontron-imx8mm/nodejs/'
+	# 	osx: 'https://www.balena.io/docs/learn/getting-started/kontron-imx8mm/nodejs/'
+	# 	linux: 'https://www.balena.io/docs/learn/getting-started/kontron-imx8mm/nodejs/'
 
 	supportsBlink: false
 
 	options: [ networkOptions.group ]
 
 	yocto:
-		machine: 'kontron-mx8mm'
```

Comment:
Hi, what I meant what that the slug name does not match.
The internal yocto machine name is a different story, and you choose whatever you want there. For example, if you do this change here I expect things break because the BSP does not define a `kontron-imx8mm` machine: https://git.kontron-electronics.de/sw/ked/meta-ked-bsp/-/tree/main/layers/meta-ked-imx/conf/machine?ref_type=heads